src/Sylius/Bundle/ResourceBundle/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 ### v0.16.0
 
 * Introduce Factory for all resources.
+* New ``DriverInterface`` for resources.
+* Reworked all listeners to use new Registry.
 
```

Comment:
Please complete the changelog. Configuration tree has been changed, validation_group -> validation_groups etc.